std/os/file.zig

```diff
@@ -357,7 +357,7 @@ pub const File = struct {
             }
             if (file_size < 0)
                 return error.Overflow;
-            return math.cast(usize, @intCast(u64, file_size));
+            return @intCast(u64, file_size);
```

Comment:
See my other comment about `windows.LARGE_INTEGER`. I think they can be handled the same way. This logic about `file_size < 0` I think would be better to be deleted. This code can assert that file_size is not negative.

Reply:
ACK